lms/resources/_js_config.py

```diff
@@ -25,9 +25,13 @@ def _h_user(self):
     def _consumer_key(self):
         return self._lti_user.oauth_consumer_key
 
-    @property
-    def _ai_getter(self):
-        return self._request.find_service(name="ai_getter")
+    def _application_instance(self):
+        """
+        Return the current request's ApplicationInstance.
+
+        :raise ConsumerKeyError: if request.lti_user.consumer_key isn't in the DB
+        """
+        return self._request.find_service(name="application_instance").get()
```

Comment:
Since this helper method now returns the actual `ApplicationInstance` (rather than the service) it can raise `ConsumerKeyError`, which it didn't before. Added the docstring for that reason